fission/src/ui/components/MainHUD.tsx

```diff
@@ -86,23 +92,23 @@ const MainHUD: React.FC = () => {
                 />
                 <div className="flex flex-col gap-0 bg-background w-full rounded-3xl">
                     <MainHUDButton
-                        value={"Settings"}
+                        value={"General Settings"}
                         icon={SynthesisIcons.Gear}
                         onClick={() => openModal("settings")}
                     />
-                    {/* <MainHUDButton
+                    {/*                     <MainHUDButton
                         value={"View"}
                         icon={SynthesisIcons.MagnifyingGlass}
                         onClick={() => openModal("view")}
                     /> */}
                     <MainHUDButton
```

Comment:
Should this be deleted?

Reply:
I'm keeping this because it will be back as soon as hunters camera and context menus are in